gaps/gap-5_payload_manifest/gap-5_payload_manifest.md

```diff
@@ -47,7 +47,7 @@ Digest algorithm used to generate manifest signature.
 
 ### `golem.srv.comp.payload.cert: String`
 
-Base64-encoded certificate in DER format.
+Base64-encoded certificate in DER, PEM, or PEM certificate chain formats.
```

Comment:
The wording is confusing for me, first I thought you added PEM twice then I realized that the third option is PEM certificate chain. Btw if certificates are supported with DER and PER encoding, then wouldn't it support certificate chains in DER encoding also?
```suggestion
Base64-encoded string of a DER or PEM encoded certificate or PEM encoded certificate chain.
```